packages/in_app_purchase/README.md

```diff
@@ -12,19 +12,19 @@ Add this to your Flutter app to:
 2. Load in app products currently owned by the user according to the underlying
    shop.
 3. Send your user to the underlying store to purchase your products.
+4. Displays a sheet that allows users to redeem subscription offer codes. (iOS 14 only)
 
 ## Getting Started
 
-This plugin is in beta. Please use with caution and file any potential issues
-you see on our [issue tracker](https://github.com/flutter/flutter/issues/new/choose).
+### Preparation
```

Comment:
I suggested adding `### Preparation` here, but another good option would be adding `## Usage` around line 51 to separate the steps for preparation and steps for actually using the plugin.

Reply:
Removed the ###preparation, added ##Usage.